source/extensions/filters/http/adaptive_concurrency/adaptive_concurrency_filter.h

```diff
@@ -34,11 +34,21 @@ class AdaptiveConcurrencyFilterConfig {
 
   bool filterEnabled() const { return adaptive_concurrency_feature_.enabled(); }
   TimeSource& timeSource() const { return time_source_; }
+  Http::Code concurrencyLimitExceededStatus() const { return concurrency_limit_exceeded_status_; }
 
 private:
+  static Http::Code toErrorCode(uint64_t status) {
```

Comment:
nit: Please move to an unnamed namespace inner function in the cc file.

Reply:
I'm not super familiar with modern C++, in fact this is the first bit of C++ I've written since undergrad. Can you clarify what you mean by an "unnamed namespace inner function" or link to an example?

Reply:
Sure. Currently `toErrorCode` is a function that is only used in the cc file, and is also defined as private (shouldn't be used by other classes).
In this case we can move the function into the cc file inside an unnamed namespace, e.g.,
```
namespace {
  Http::Code toErrorCode(...) {
    ...
  }
}
```

Example can be seen [here](https://github.com/envoyproxy/envoy/blob/a2b595da4fd48c7c6fa399637f5001d0c0f0f6a5/source/extensions/filters/http/oauth2/config.cc#L25-L38) where `secretsProvider()` is defined only internally in this file.